README.md

```diff
@@ -166,8 +154,7 @@ compiled with appropriate `target_feature`s, so this cannot occur.
 Benchmarks are run using [`criterion.rs`][criterion]:
 
 ```sh
-cargo bench --no-default-features --features "std u32_backend"
-cargo bench --no-default-features --features "std u64_backend"
+cargo bench --no-default-features --features std
 # Uses avx2 or ifma only if compiled for an appropriate target.
 export RUSTFLAGS="-C target_cpu=native"
 cargo bench --no-default-features --features "std simd_backend"
```

Comment:
`cargo bench --features "simd_backend"` will suffice

Reply:
FYI, running that I notice that backend no longer compiles, but it's unrelated to this PR.

Reply:
Fixed in f0bff96